public/src/components/channelManagement/campaigns/TestCard.tsx

```diff
@@ -84,24 +84,24 @@ const useStyles = makeStyles(({ spacing }: Theme) => ({
 }));
 
 interface TestCardProps {
-  test: CombinedTest;
+  test: Test;
   keyId: string;
   linkPath: string;
 }
 
 function TestCard({ test, keyId, linkPath }: TestCardProps): React.ReactElement {
   const classes = useStyles();
 
-  const getVariantNames = (variants: CombinedVariant[]) => {
+  const getVariantNames = (variants: Variant[]) => {
     if (variants.length > 0) {
       return (
         <div className={classes.variantsData}>Variants: {variants.map(v => v.name).join(', ')}</div>
       );
     }
-    return <div className={classes.dataWarning}>No variants have been created for this test!</div>;
+    return <div className={classes.dataWarning}>No Variants have been created for this Test.</div>;
```

Comment:
was this accidentally changed from `No variants have been created for this test!`?

Reply:
Again - deliberate, following on from Mael's suggestion.